app/pages/[teamSlug]/reports/report.graphql

```diff
@@ -0,0 +1,48 @@
+query report($projectId: ID!, $startDate: Date!, $endDate: Date!) {
+  report(projectId: $projectId, from: $startDate, to: $endDate) {
```

Comment:
Let's keep it consistent and use the variable names `$from` and `$to`.

Reply:
OK, and thanks for the review!